src/workerd/api/analytics-engine-test.wd-test

```diff
@@ -0,0 +1,23 @@
+using Workerd = import "/workerd/workerd.capnp";
+
+const analyticsWorker :Workerd.Worker = (
+    modules  = [
+      (name = "worker", esModule = "import * as assert from 'node:assert'; export default { async fetch(request) { let val = await request.json(); console.log(JSON.stringify(val)); assert.deepStrictEqual(val['dataset'], [97,110,97,108,121,116,105,99,115]); assert.deepStrictEqual(val['double1'], 25); assert.deepStrictEqual(val['blob1'], [84,101,115,116,66,108,111,98]); return new Response('Hello world'); }, };"),
```

Comment:
Nit: why are you embedding the script directly? Would be more manageable to put this in a separate js file?

Reply:
The way the `wd_test` bazel target is set up right now it can only have one js file ending in `-test.js`. I couldn't figure out how to add another file so I decided to embed it